packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseWalletConfirmationStep1.tsx

```diff
@@ -0,0 +1,80 @@
+import { Rows, Card, Columns, Button, Text } from '@trezor/components';
+import { HELP_CENTER_PASSPHRASE_URL } from '@trezor/urls';
+import { Translation } from 'src/components/suite/Translation';
+import { TrezorLink } from 'src/components/suite/TrezorLink';
+import { PassphraseHeading } from './PassphraseHeading';
+import { ContentType } from './types';
+
+type PassphraseWalletConfirmationStep1Props = {
+    setContentType: React.Dispatch<React.SetStateAction<ContentType>>;
+    onCancel: () => void;
+};
+
+export const PassphraseWalletConfirmationStep1 = ({
+    setContentType,
+    onCancel,
+}: PassphraseWalletConfirmationStep1Props) => {
+    return (
```

Comment:
nit: inline return `=> (<> ....)` with no `{` and no `return`

I thought we have a lint for this. :thinking:

Reply:
I think we don't have the rule (yet) 😅